cmd/internal/flag.go

```diff
@@ -114,9 +114,9 @@ func initRegistryClientFlags() {
 }
 
 func initImageVerifyCacheFlags() {
-	flag.BoolVar(&imageVerifyCacheEnabled, "imageVerifyCacheEnabled", true, "Whether to use a TTL cache for storing verified images.")
-	flag.Int64Var(&imageVerifyCacheMaxSize, "imageVerifyCacheMaxSize", 1000, "Max size limit for the TTL cache, 0 means default 1000 size limit.")
-	flag.DurationVar(&imageVerifyCacheTTLDuration, "imageVerifyCacheTTLDuration", 60*time.Minute, "Max TTL value for a cache, 0 means default 1 hour TTL.")
+	flag.BoolVar(&imageVerifyCacheEnabled, "imageVerifyCacheEnabled", true, "Whether to use a TTL cache for verifying images.")
+	flag.Int64Var(&imageVerifyCacheMaxSize, "imageVerifyCacheMaxSize", 1000, "Maximum number of keys that can be stored in the TTL cache. Keys are a combination of policy elements along with the image reference. Default is 1000. 0 sets the value to default.")
```

Comment:
> `Keys are a combination of policy elements along with the image reference`

Is this relevant to users?

Do we need a valid range for these settings? What if someone sets to 1?

Reply:
The description change was to distinguish between a key used to literally verify an image and a key as a hash. There should be some explanation as to what "1000" means. 1000 what? Bytes? Kilobytes? Something else? Maybe it could be "1000 entries" or something similar, but it still needs explanation at least in the docs.

Reply:
> Do we need a valid range for these settings?

I don't think we should limit the range of values this flag can have, since Ristretto allows all values, so should we, If the user knows exactly what the flag does and wants to set it to 1,they can and its fine (In case of 1 the new entry will replace the existing entry)

Reply:
I agreed with @vishal-chdhry, the user should be able to configure whatever the desired entries for the cache. @vishal-chdhry - do you know the estimate of memory usage with 1000 entries?

I say we keep it unbounded unless we know the exact limitation with it.

Reply:
@realshuting every entry takes around 30Bytes, so `1000` entries should take around `30KB`